config/locales/clearance.en.yml

```diff
@@ -0,0 +1,59 @@
+---
```

Comment:
Great use of the `locales` feature, but let's remove the parts that we're not using, so it's easier for us when we need to edit the parts that we are. WDYT? 😀